opencti-platform/opencti-front/src/utils/utils.ts

```diff
@@ -32,3 +32,9 @@ export const deleteElementByValue = (obj: Record<string, string>, val: string) =
   }
   return obj;
 };
+
+export const getUri = (id: string) => {
```

Comment:
Maybe rename the method to `getFileUri`" ? Otherwise `getUri` is not very explicit I think